src/Card/index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import './index.css';
 
-const Card = ({caption, text}) => (
-        <div className="card">
-            <p>{caption}</p>
+const Card = ({caption, text, checked, checkboxChange}) => (
+        <div className={'card ' + (checked ? 'border' : '')}>
```

Comment:
You would possibly like to use special css package that allows to easily handle conditional styling:
https://www.npmjs.com/package/classnames